apiary.apib

```diff
@@ -3022,6 +3022,9 @@ Update a tasks.
         + id: `00ed6266-a5bd-4aac-a292-2582017b6400` (string, required)
         + description (string, optional)
         + due_at: `2016-02-04T16:44:33+00:00` (string, optional)
+        + work_type_id: `32665afd-1818-0ed3-9e18-a603a3a21b95` (string, optional)
+        + estimated_duration: 3600 (number, optional)
```

Comment:
PP: the field `estimated_duration` does not explain the measure: is it seconds, hours, days? I would rename it to `estimated_duration_seconds`

Reply:
Of course, it would need to be added in tasks.apib, not her :D

Reply:
good point 👌